tests/test_prod_smoketest.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+"""
+A prod test for the DSS, checks core API Requests to know whats available. Works with data present.
+"""
+import os, sys, argparse, time, uuid, json, shutil, tempfile, unittest
+import subprocess
+
+import boto3
+import botocore
+from cloud_blobstore import BlobStore
+from itertools import product
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from tests.infra import testmode
+from tests.smoktest.base_smoketest import BaseSmokeTest, run, run_for_json
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("--no-clean", dest="clean", action="store_false",
+                    help="Don't remove the temporary working directory on exit.")
+args = argparse.Namespace(clean=True)
+
+
+@testmode.integration
+class ProdSmoketest(BaseSmokeTest):
+    params = BaseSmokeTest.params
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.url = "https://www.example.com"
+        cls.query = {"query": {"bool": {"must": [{"term": {"admin_deleted": "true"}}]}}}
+
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
Can this method be moved into the base class?

Reply:
changed